### PR TITLE
expansion packs. booyah!

### PR DESCRIPTION
**Pull request by colestrode**
Expansion packs, toggled on and off using setMode(). Bam!

Expasion packs 2 and 3 are already included, I added 4 and 6, but couldn't find 5 online. I split out our own stuff into a separate deck while I was at it.

### CODE REVIEW

**Comment by jasonrhodes** [comment]
ahem...

deck.speck.js

File: src/game.js [hunk 1]
```diff
@@ -24,24 +25,29 @@ var hasProp = {}.hasOwnProperty;
...
-  this.shuffle();
+  this.resetDecks();
 };
 
 Game.prototype.toggle_mode = function (mode) {
   this.db.modes = this.db.modes || {};
   this.db.modes[mode] = !this.db.modes[mode];
+  deck.setModes(this.db.modes);
+
+  this.resetDecks(); // adds/removes cards for toggled mode
```
Comment (jasonrhodes) [inline src/game.js:49]: So any time you turn a mode on/off it re-shuffles all the decks in play, right?

**Comment by jasonrhodes** [comment]
I rebased for you (god i love git so much), so merge conflicts are gone :boom:

File: src/game.js (hunk 1 above)
Comment (colestrode) [inline src/game.js:49]: Yep. There is a chance that a previously played card could show up again soon. The alternative would be to finish the existing deck, then the new set would get pulled in. I like the immediate gratification 😄

**Comment by colestrode** [comment]
speck is gone. also, i changed the name of the file.

**Comment by colestrode** [comment]
one day, I will need to learn this rebasing wizardry you speak of.

**Comment by max-mathieu** [comment]
I think the code reads how many _____ are in the question to know how many cards can played.
Question like the first one of the expansion 4 don't have one for instance. Not sure about the best fix

**Comment by jasonrhodes** [comment]
Good call, see #28

**Pull request merged**